packages/from-query/src/generate.ts

```diff
@@ -66,7 +66,20 @@ export default (OPTIONS: IFromQueryOptions): (ir: IOperation) => string => {
     }
   };
 
-  const printField: (name: string, type: TypeDefinition | string, node: Selection, nameOverride?: string) => string = (name, type, node, nameOverride) => `${name}: ${printType(type, node, nameOverride)};`;
+  const hasDirectives: (directives: IDirectiveMap) => boolean = directives =>
```

Comment:
can we rename this to convey that it checks for specific directives instead of _any_ directive as the name would suggest?

something like `hasOptionalDirective` or so?